packages/zoe/src/contracts/constantProduct/README.md

```diff
@@ -0,0 +1,139 @@
+# Constant Product AMM
+
+A constant product automatic market maker based on our Ratio library. It charges
+two kinds of fees: a pool fee remains in the pool to reward the liquidity
+providers and a protocol fee is extracted to fund the economy.
+
+This algorithm uses the x*y=k formula directly, without fees. Briefly, there are
+two kinds of assets, whose values are kept roughly in balance through the
+actions of arbitrageurs. At any time, a trader can come to the pool and offer to
```

Comment:
Hmm, the way this is written doesn't make it clear a trade is intended. It sounds like adding liquidity at first.